string NAME=aligned

ObjectLocation l0
ObjectLocation l1
